### PR TITLE
Dont exit if 'mount.glusterfs -V' resulted in an error.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by rootfs** [comment]
@k8s-bot ok to test

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -379,51 +379,51 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
...
-	parseStr := string(cmdOut)
-	if parseStr != "" {
-
-		//Store the version line from parseStr
-		mountStrSlice := dstrings.Split(parseStr, "\n")
-		if len(mountStrSlice) > 0 {
-			mountStr = mountStrSlice[0]
-		}
-		if mountStr != "" {
+		glog.Warningf("Failed to get binary '%s' version", linuxGlusterMountBinary)
```
Comment (rootfs) [inline pkg/volume/glusterfs/glusterfs.go]: `'%s'` -> %q

Comment (humblec) [inline pkg/volume/glusterfs/glusterfs.go]: Done.

File: pkg/volume/glusterfs/glusterfs.go [hunk 2]
```diff
@@ -379,51 +379,51 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
...
-	parseStr := string(cmdOut)
-	if parseStr != "" {
-
-		//Store the version line from parseStr
-		mountStrSlice := dstrings.Split(parseStr, "\n")
-		if len(mountStrSlice) > 0 {
-			mountStr = mountStrSlice[0]
-		}
-		if mountStr != "" {
+		glog.Warningf("Failed to get binary '%q' version", linuxGlusterMountBinary)
```
Comment (rootfs) [inline pkg/volume/glusterfs/glusterfs.go]: `'%q'` -> `%q`
%q	a double-quoted string safely escaped with Go syntax

**Comment by humblec** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

**Comment by humblec** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

**Comment by thockin** [comment]
@rootfs - I just wanted to say thansk for continuing to be all over these PRs :)

**Comment by humblec** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

**Pull request merged**